### ISSUE TITLE
No 'good first issues'.

### ISSUE THREAD

**Issue by p4puniya** [Giskard-AI/giskard#271]
## No Good First Issues.

As a budding community, it's necessary to attract the new talent towards one's organisation. And 'G.F.I.' is the best practice to do that. But I didn't find any such issues. 

## 📎 Additional context

A few good first issues can be:
- Modifying the documentations.
- Updating tags such as version, Licence etc in the readme.md and other documents whenever there's a new update. 
- Adding hyperlinks to the necessary models (such as PyTorch, TensorFlow etc) to make the docs more interactive.
- Adding operating demos about a particular feature.
- Other issues that are time consuming but simple and easy.
As long as there's something for the newcomers to do... The community will grow.

**Comment by andreybavt** [comment]
Now that we've migrated your tasks to GitHub, we'll be adding [d:easy](https://github.com/Giskard-AI/giskard/labels/d%3Aeasy) tag to mark easy tasks that could be good starting points for anyone who wants to contribute to Giskard
